### DISCUSSION TITLE
Expedition #2: Beachhead - Starting System incompatible with Freighter Warp

### DISCUSSION THREAD

**Discussion by rgoodeno** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#40] [General]
Not sure if this was unique just to my start but I had the following issue with Expedition 2: Beachhead:

1. Started in a red stellar system
2. Freighter must be warped in
3. Freighter does not contain required tech module to warp to red systems
4. Starting ship does not contain a hyperdrive
5. Purchaseable and crashed ships in region do not have hyperdrives
6. Space station teleporter will not allow TP to freighter until it is called in the first time
7. Anomaly cannot be accessed to learn hyperdrive blueprint until after first warp
8. Other freighters do not seem to spawn in first system

I got fortunate in finding a blackhole anomaly which brought me to a yellow system and allowed me to progress enough to get the red upgrade for the freighter, TP back, and warp it in but others may not be so lucky if they have the same issue I did.

**Comment by cwmonkey** [comment]
Fixed in 524fa9d.

You can grab the new json from https://cwmonkey.github.io/nms-expeditions/ or this repo.

The yellow star fix and tech slot fix will only work for new games/starting a new expedition, but the Phase 5, milestone 4 and 5 fix will work for existing expedition saves.
